src/pubsub.js

```diff
@@ -17,14 +19,24 @@ class BasicPubSub extends Pubsub {
    * @param {String} debugName
    * @param {String} multicodec
    * @param {Object} libp2p libp2p implementation
+   * @param {Object} options
+   * @param {bool} options.fallbackToFloodsub if dial should fallback to floodsub, defaults to true
    * @constructor
    */
-  constructor (debugName, multicodec, libp2p) {
+  constructor (debugName, multicodec, libp2p, options) {
     super(debugName, multicodec, libp2p)
```

Comment:
Now that we're taking options here, we should pass them to Pubsub so we can pass them down.

```suggestion
     super(debugName, multicodec, libp2p, options)
```